doc/liblouis.texi

```diff
@@ -835,6 +851,99 @@ numericmodechars .,
 midendnumericmodechars -/
 @end example
 
+Note: This opcode was implemented in Liblouis specifically in order
+to support Unified English Braille (UEB). It may be used in any table,
+but may have unpredicted side-effects if used outside the intended
+context. Use with great care, and test thoroughly.
+
+@end table
+
+@node Opcodes for Standing Alone Sequences
+@section Opcodes for Standing Alone Sequences
+
+The term "standing alone" comes from the specification of Unified
+English Braille (UEB). In Liblouis, a letter or letters-sequence is
+considered to be "standing alone" if it is preceded and followed by a
+space, and/or other characters that you choose as delimiters, e.g.
+@samp{-}. A "standing alone" sequence can be thought of as a word in
+a very broad sence. With the opcodes described in this section, you
+can decide what characters constitute a delimiter, and what characters
+can attach to the beginning or end of a word or standing alone
+sequence.
+
+Note: The opcodes in this section were implemented in Liblouis
+specifically in order to support Unified English Braille (UEB). they
+may be used in any table, but may have unpredicted side-effects if
+used outside the intended context. Use with great care, and test
+thoroughly.
+
+@table @code
+@opcode{seqdelimiter, <characters>}
+All the characters listed with this opcode designate a valid beginning
+and ending to a letter sequence used to determine when a letter
+sequence is “standing alone”. This again determines whether
+ word contractions, part word contractions or nocontractsign should
+ be applied..
+
+Spaces do not need to be listed as
+they are automatically delimiters.
+
+For example, in UEB (section 2.6.1 page 15), any hyphen or dash count
+as delimiters.
+
+This opcode can be used several times, but the characters must have
+already been defined.
+
+Example:
+
+@example
+seqdelimiter -—
+@end example
+
+@opcode{seqbeforechars, <characters>}
+Characters specified with this opcode may appear between a beginning
+sequence delimiter and the letter sequence itself.
+
+For example, in UEB (2.6.2, page 15), opening parenthesis and opening
+quotations and such are allowed.
+
+This opcode can be used several times, but the characters must have
+already been defined.
+
+Example:
+@example
+seqbeforechars ([@{"“'‘
```

Comment:
These chars have special meaning to texinfo, so they need to be quoted, see [Special Characters: Inserting @ {} , \ # &](https://www.gnu.org/software/texinfo/manual/texinfo/html_node/Special-Characters.html)

Reply:
ah, sorry I believe you already took care  of the quoting